File: KoRadio/UI/ko_radio_mobile/lib/models/service.dart

import 'package:json_annotation/json_annotation.dart';
part 'service.g.dart';
@JsonSerializable()
class Service{
  int serviceId;
  String serviceName;
  String image;
  bool isDeleted;


  
  Service({required this.serviceId, required this.serviceName, required this.image, required this.isDeleted});

  factory Service.fromJson(Map<String, dynamic> json) => _$ServiceFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$ServiceToJson(this);
}

File: KoRadio/KoRadio.Model/Service.cs
using System;
using System.Collections.Generic;

namespace KoRadio.Model
{




    public  class Service
    {
		public int ServiceId { get; set; }

		public string ServiceName { get; set; } = null!;

		public byte[] Image { get; set; } = null!;

		public bool IsDeleted { get; set; }
		public int FreelancerCount { get; set; }
		public int CompanyCount { get; set; }

	}
}
